source/release-notes.txt

```diff
@@ -10,6 +10,17 @@ Release Notes
    :depth: 1
    :class: twocols
 
+|compass| 1.34.2
+----------------
+
+*Released December 16, 2022*
+
+Bug Fixes:
+
+- Fix (compass-editors): fix nested field autocomplete :issue:`COMPASS-6335`
+- Fix (schema): pass the preference as a prop when nesting Field 
```

Comment:
That’s not-so-great commit naming on our side :) This could be something like 
```suggestion
- Fix (schema): fix display of geo visualizations for nested fields 
```

Reply:
Updated, thanks!